src/cli.ts

```diff
@@ -87,6 +87,10 @@ const parsed = yargs(hideBin(process.argv))
     boolean: true,
     default: true,
   })
+  .option('backup-before-storage-update', {
```

Comment:
Backups will error out if we are using askar-postgres, so we should not allow this option to be true if postgres is being used